controllers/configuration_controller.go

```diff
@@ -243,8 +242,15 @@ func (r *ConfigurationReconciler) reconcile(ctx context.Context, obj *v1alpha1.C
 	}
 
 	// create snapshot
-	snapshotName := fmt.Sprintf("%s/snapshots/%s:%s", r.OCIRegistryAddr, obj.Spec.SnapshotTemplate.Name, obj.Spec.SnapshotTemplate.Tag)
-	snapshotDigest, err := r.writeSnapshot(ctx, snapshotName, artifactPath.Name())
+	snapshotName := fmt.Sprintf(
+		"%s/%s/%s/%s/%s",
+		r.OCIRegistryAddr,
+		componentVersion.Spec.Component,
+		componentVersion.Status.ReconciledVersion,
+		obj.Spec.SnapshotTemplate.Name,
+		obj.Spec.SnapshotTemplate.Tag,
```

Comment:
why `name` and `tag` in the path? Is the tag considered a different resource?

Reply:
Tag is the version. So yes, I guess. :)